treeherder/model/fixtures/repository.json

```diff
@@ -1145,7 +1145,7 @@
         "name": "go-bouncer",
         "url": "https://github.com/mozilla-services/go-bouncer",
         "branch": "master",
-        "active_status": "active",
+        "active_status": "onhold",
```

Comment:
This was newly added but isn't getting new pushes - @davehunt might need to add a webhook to the repo, since looking at https://github.com/mozilla-services/go-bouncer there were pushes 3 days ago.

It's also causing "Missing push" errors on New Relic since jobs are being submitted but are having to be skipped since no push with which to associate them.